vulnerabilities/management/commands/import.py

```diff
@@ -102,6 +102,13 @@ def _import_data(self, importers, cutoff_date):
         for importer in importers:
             self.stdout.write(f'Importing data from {importer.name}')
             batch_size = int(getattr(self, 'batch_size', 10))
-            ImportRunner(importer, batch_size).run(cutoff_date=cutoff_date)
-            self.stdout.write(
-                self.style.SUCCESS(f'Successfully imported data from {importer.name}'))
+            try:
+                ImportRunner(importer, batch_size).run(cutoff_date=cutoff_date)
+                self.stdout.write(
+                    self.style.SUCCESS(f'Successfully imported data from {importer.name}'))
+
+            except Exception as e:
+                # TBD
+                self.stdout.write(
```

Comment:
IMHO just writing to stdout is not good enough as the likeliness we will ever look at logs is low.
This should be written to the DB instead.

Reply:
I am reverting this. This seems out of scope for this PR.